Sources/joyfill-components-swift/Components/Table/ViewTable.swift

```diff
@@ -350,6 +389,18 @@ public class ViewTable: UIViewController, TextFieldCellDelegate, TextViewCellDel
     
     // Action for close button
     @objc func clossTapped() {
```

Comment:
I think you meant `closeTapped` and not clossTapped

Reply:
Done